CollabApp/CollabApp.mvc/Views/Home/Chat.cshtml

```diff
@@ -12,10 +12,6 @@
     <h1>Welcome to the chat!</h1>
 
     <div class="container">
-        <div class="row">
-            <div class="col-1">User</div>
```

Comment:
now i can send only empty messages
[cinnamon-2023-09-30T164751+0300.webm](https://github.com/niasenago/PSI-project/assets/118674771/8dd7519a-e4aa-43c1-9154-ae74d17cbded)

Reply:
Let's hope this is the last bug

Reply:
bug is solved